### ISSUE TITLE
Missing VCF header lines

### ISSUE THREAD

**Issue by eldariont** [smehringer/SViper#5]
Hi Svenja,

bcftools is complaining about missing VCF header lines in SViper's VCF output:

```
[W::vcf_parse] INFO 'SEQ' is not defined in the header, assuming Type=String
[W::vcf_parse] FILTER 'FAIL5' is not defined in the header
[W::vcf_parse] FILTER 'FAIL3' is not defined in the header
[W::vcf_parse] FILTER 'FAIL2' is not defined in the header
[W::vcf_parse] FILTER 'FAIL4' is not defined in the header
[E::bcf_write] Unchecked error (2), exiting
```

Could you maybe add these?

Cheers
David

**Comment by smehringer** [comment]
Sure I'll have a look at it next week!

**Comment by smehringer** [comment]
Hey @eldariont,

please check out the current master, build and run SViper again and tell me if it works with the new [commit](https://github.com/smehringer/SViper/commit/eeb183e19d0dc076c68eda08cbecdf6e565d93ee)

I include the info SEQ field in case it does not appear in the header.
I always include the FAIL filter tags, since I assume nobody else uses them.

This actually caught a tiny bug also! FILTER1 and 4 where ambiguous so now there are also FAIL6 and 7 although they should seldom occur since they refer to an unknown reference name of the variant. (see [this commit](https://github.com/smehringer/SViper/commit/2b9664724937768c59189ca1593a695f148e6d30) as a reference)

**Comment by eldariont** [comment]
Thanks for working on this.
There remains one small problem that bcftools still complains about. The spec requires the ID of a meta-information field to not be surrounded by quotation marks.

So it needs to be
`output_vcf << "##FILTER=<ID=FAIL1,Description=\"No long reads in variant region.\">" << std::endl;`
instead of
`output_vcf << "##FILTER=<ID=\"FAIL1\",Description=\"No long reads in variant region.\">" << std::endl;`.

**Comment by smehringer** [comment]
:roll_eyes: :roll_eyes: fine. thanks for the info!

**Comment by eldariont** [comment]
Yes, it's annoying. I can also create a PR if you prefer.

**Comment by smehringer** [comment]
sure why not, thanks! :)

**Comment by smehringer** [comment]
Fixed by #7
